src/utils.ts

```diff
@@ -0,0 +1,37 @@
+import { readdir } from "fs/promises";
+import { RowArray, RowObject } from "./types";
+
+export const jsonDataPath = "./src/data/json";
+export const csvDataPath = "./src/data/csv";
+
+export const readDataFiles = async (
+  path = jsonDataPath
+): Promise<string[] | null> => {
+  try {
+    const files = await readdir(path);
+    return files;
+  } catch (error) {
+    console.error(error);
+    return null;
+  }
+};
+
+export const isJestRunning = (): boolean =>
```

Comment:
I am checking if program is running in test only to prevent creating csv directory in src folder when program starts. That's becouse I don't want jest to affect program at all.